### PR TITLE
When TEST_CPP is set to no, disable all tests

### PR DESCRIPTION
**Pull request by isuruf**
This script is used by other repos for installation as well and there's no need to run tests on the main repo

### CODE REVIEW

File: bin/test_travis.sh [hunk 1]
```diff
@@ -104,4 +109,4 @@ if !  egrep " $" -nr --include=\*.{cpp,h,inc}  --exclude-dir=*{teuchos,/build/}*
 else
     exit -1;
 fi
-# TODO: Add similar grep checks for space after comma,, space after `if`, space between `)` and `{` also
\ No newline at end of file
+# TODO: Add similar grep checks for space after comma,, space after `if`, space between `)` and `{` also
```
Comment (abinashmeher999) [inline bin/test_travis.sh]: I think you missed a newline at the end.

Comment (isuruf) [inline bin/test_travis.sh]: I added a newline

Comment (abinashmeher999) [inline bin/test_travis.sh]: Oops, my bad.

**Comment by certik** [comment]
+1

**Pull request merged**